exercises/isogram/src/example/java/Isogram.java

```diff
@@ -0,0 +1,29 @@
+package exemple;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Isogram {
+	
+	public static boolean isIsogram(String word){
+		
+		Set<Character> charset= new HashSet<Character>();
```

Comment:
You can just use the [diamond operator](http://stackoverflow.com/questions/4166966/what-is-the-point-of-the-diamond-operator-in-java-7) for your `HashSet` declaration like this:

`Set<Character> charset = new HashSet<>();`